### PR TITLE
⚠️ CONFLICT! Lineage pull request for: skeleton

### PR DESCRIPTION
**Pull request by cisagovbot**
# Lineage Pull Request: CONFLICT #

[Lineage] has created this pull request to incorporate new changes found in an
upstream repository:

Upstream repository: [`https://github.com/cisagov/skeleton-python-library.git`](https://github.com/cisagov/skeleton-python-library.git)
Remote branch: `HEAD`

Check the changes in this pull request to ensure they won't cause issues with
your project.

The `lineage/skeleton` branch has **one or more unresolved merge conflicts**
that you must resolve before merging this pull request!

## How to resolve the conflicts ##

1. Take ownership of this pull request by removing any other assignees.

1. Clone the repository locally, and reapply the merge:

    ```console
    git clone git@github.com:cisagov/pe-reports.git pe-reports
    cd pe-reports
    git remote add skeleton https://github.com/cisagov/skeleton-python-library.git
    git remote set-url --push skeleton no_push
    git switch develop
    git checkout -b lineage/skeleton --track origin/develop
    git pull skeleton HEAD
    git status
    ```

1. Review the changes displayed by the `status` command.  Fix any conflicts and
   possibly incorrect auto-merges.

1. After resolving each of the conflicts, `add` your changes to the
   branch, `commit`, and `push` your changes:

    ```console
    git add .github/CODEOWNERS 
    git commit
    git push --force --set-upstream origin lineage/skeleton
    ```

    Note that you may *append* to the default merge commit message
    that git creates for you, but *please do not delete the existing
    content*.  It provides useful information about the merge that is
    being performed.

1. Wait for all the automated tests to pass.

1. Confirm each item in the "Pre-approval checklist" below.

1. Remove any of the checklist items that do not apply.

1. Ensure every remaining checkbox has been checked.

1. Mark this draft pull request "Ready for review".

## ✅ Pre-approval checklist ##

Remove any of the following that do not apply. If you're unsure about
any of these, don't hesitate to ask. We're here to help!

- [x] ✌️ The conflicts in this pull request have been resolved.
- [x] All relevant type-of-change labels have been added.
- [x] All new and existing tests pass.

## ✅ Pre-merge checklist ##

Remove any of the following that do not apply. These boxes should
remain unchecked until the pull request has been approved.

- [ ] Bump major, minor, patch, or pre-release version [as
      appropriate](https://semver.org/#semantic-versioning-specification-semver)
      via the `bump_version.sh` script *if* this repository is
      versioned *and* the changes in this PR [warrant a version
      bump](https://semver.org/#what-should-i-do-if-i-update-my-own-dependencies-without-changing-the-public-api).

## ✅ Post-merge checklist ##

Remove any of the following that do not apply.

- [ ] Create a release.

---

**Note:** *You are seeing this because one of this repository's maintainers has
configured [Lineage] to open pull requests.*

For more information:

🛠 [Lineage] configurations for this project are stored in `.github/lineage.yml`

📚 [Read more about Lineage][Lineage]

[//]: # (lineage:metadata:{&quot;slayed&quot;:true})
[Lineage]: https://github.com/cisagov/action-lineage/ "Lineage GitHub Action"

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -3,8 +3,8 @@
 # These owners will be the default owners for everything in the
 # repo. Unless a later match takes precedence, these owners will be
 # requested for review when someone opens a pull request.
-* @aloftus23 @cduhn17 @dav3r @DJensen94 @elr64 @felddy @jsf9k @mcdonnnj @schmelz-ctr @stewartl97
+* @aloftus23 @cduhn17 @dav3r @DJensen94 @elr64 @felddy @jasonodoom @jsf9k @mcdonnnj @schmelz-ctr @stewartl97
```
Comment (jsf9k) [inline .github/CODEOWNERS]: GitHub flagged @elr64 and @schmelz-ctr as "unknown owners", implying that either they no longer exist as GitHub users or no longer have write access to this repository.  I suspect that @elr64 no longer has write access, while @schmelz-ctr no longer exists at all, and if so then both should be removed.

**Review by jsf9k — CHANGES_REQUESTED** [review]
Please address the codeowners issue I brought up.

File: .github/CODEOWNERS (hunk 1 above)
Comment (dav3r) [reply .github/CODEOWNERS]: @cduhn17 or @aloftus23 - can you please confirm that elr64 and schmelz-ctr should be removed as code owners here or point us in the direction of someone who can?

Comment (jsf9k) [reply .github/CODEOWNERS]: @cduhn17 and @aloftus23 - Bumping this for visibility.

Comment (cduhn17) [reply .github/CODEOWNERS]: @dav3r , 
Yes you can remove @schmelz21 and @elr64 as code owners as they have moved on from P&E

Comment (jsf9k) [reply .github/CODEOWNERS]: Removed in commit a5acb81.

**Review by jsf9k — APPROVED** [review]

**Review by dav3r — APPROVED** [review]

**Review by dv4harr10 — APPROVED** [review]

**Pull request merged**